src/pages/Tutorial.scss

```diff
@@ -4,18 +4,37 @@
     --border-color: transparent;
   }
 
-  .swiper-slide {
-    display: block;
-  }
-
   .slide-title {
     margin-top: 2.8rem;
   }
 
+  .slider {
+    display: grid;
+    grid-template-columns: repeat(4, 100%);
+    grid-template-rows: 1fr;
+
+    height: 100%;
+
+    overflow: scroll;
+    scroll-snap-type: x mandatory;
```

Comment:
Can we add `scroll-snap-stop` you can't accidentally swipe over multiple slides: https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-snap-stop